CONTRIBUTING.md

```diff
@@ -5,66 +5,78 @@ First off, thanks for taking the time to contribute! :rocket::rocket::rocket:
-    - [Your First Code Contribution](#your-first-code-contribution)
-    - [Pull Requests](#pull-requests)
-  - [Styleguides](#styleguides)
-    - [Git Commit Messages](#git-commit-messages)
-    - [C++ Styleguide](#c-styleguide)
-      - [File Formatting](#file-formatting)
-      - [Class Structure](#class-structure)
-      - [Method Structure](#method-structure)
-      - [Coding Rules](#coding-rules)
-        - [Acronyms](#acronyms)
-        - [Types](#types)
-      - [Syntax](#syntax)
-      - [Idioms](#idioms)
-        - [Named Constructor](#named-constructor)
-        - [PIMPL](#pimpl)
-    - [Python Styleguide](#python-styleguide)
-    - [Documentation Styleguide](#documentation-styleguide)
-  - [Additional Notes](#additional-notes)
-    - [Issue and Pull Request Labels](#issue-and-pull-request-labels)
-      - [Type of Issue and Issue State](#type-of-issue-and-issue-state)
-  - [References](#references)
-
-## Code of Conduct
+##  1. <a name='TableOfContents'></a>Table Of Contents
+
+<!-- vscode-markdown-toc -->
+* 1. [Table Of Contents](#TableOfContents)
+* 2. [Code of Conduct](#CodeofConduct)
+* 3. [I Have a Question!](#IHaveaQuestion)
+* 4. [How Can I Contribute?](#HowCanIContribute)
+	* 4.1. [Reporting Bugs](#ReportingBugs)
+		* 4.1.1. [Before Submitting A Bug Report](#BeforeSubmittingABugReport)
+		* 4.1.2. [How Do I Submit A (Good) Bug Report?](#HowDoISubmitAGoodBugReport)
+	* 4.2. [Suggesting Enhancements](#SuggestingEnhancements)
+		* 4.2.1. [How Do I Submit A (Good) Enhancement Suggestion?](#HowDoISubmitAGoodEnhancementSuggestion)
+	* 4.3. [Your First Code Contribution](#YourFirstCodeContribution)
+	* 4.4. [Pull Requests](#PullRequests)
+	* 4.5. [Git Flow](#GitFlow)
+	* 4.6. [Commit Messages](#CommitMessages)
+* 5. [Styleguides](#Styleguides)
+	* 5.1. [C++ Styleguide](#CStyleguide)
+		* 5.1.1. [File Formatting](#FileFormatting)
+		* 5.1.2. [Class Structure](#ClassStructure)
+		* 5.1.3. [Method Structure](#MethodStructure)
+		* 5.1.4. [Coding Rules](#CodingRules)
+		* 5.1.5. [Code Format](#CodeFormat)
+		* 5.1.6. [Idioms](#Idioms)
+	* 5.2. [Python Styleguide](#PythonStyleguide)
+		* 5.2.1. [Python Bindings](#PythonBindings)
+	* 5.3. [Documentation Styleguide](#DocumentationStyleguide)
+* 6. [Development Setup](#DevelopmentSetup)
+	* 6.1. [Development Environment](#DevelopmentEnvironment)
+		* 6.1.1. [Development Across OSTk Libraries](#DevelopmentAcrossOSTkLibraries)
+	* 6.2. [Build](#Build)
+	* 6.3. [Test](#Test)
+	* 6.4. [Python](#Python)
+	* 6.5. [JupyterLab](#JupyterLab)
+	* 6.6. [Packages](#Packages)
+* 7. [Additional Notes](#AdditionalNotes)
+	* 7.1. [Issue and Pull Request Labels](#IssueandPullRequestLabels)
+		* 7.1.1. [Type of Issue and Issue State](#TypeofIssueandIssueState)
+* 8. [References](#References)
+
+<!-- vscode-markdown-toc-config
+	numbering=true
+	autoSave=true
+	/vscode-markdown-toc-config -->
+<!-- /vscode-markdown-toc -->
```

Comment:
needed? TBR

Reply:
I believe so, again it goes with the auto-generated TOC format

Reply:
fixed